MinimOsd_Extra/protocols/NMEA.h

```diff
@@ -54,17 +54,18 @@ uint32_t GPS_coord_to_degrees(char* s) {
     min += DIGIT_TO_VAL(*q++);
   }
   // convert fractional minutes
-  // expect up to four digits, result is in
+  // expect up to four (five) digits, result is in
   // ten-thousandths of a minute
   if (*p == '.') {
     q = p + 1;
-    for (i = 0; i < 4; i++) {
+    for (i = 0; i < 5; i++) {
       frac_min *= 10;
       if (isdigit(*q))
         frac_min += *q++ - '0';
     }
   }
-  return deg * 10000000UL + (min * 1000000UL + frac_min*100UL) / 6;
+//  return deg * 10000000UL + (min * 1000000UL + frac_min*100UL) / 6; //this is wrong
+  return deg * 10000000UL + (min * 10000000UL + frac_min * 100UL) / 60;
 }
```

Comment:
here you tries to correct your change to 5 digits but DISCARDS last one because coords in integer. So integral results of this changes is ZERO. And what's the point?